templates/cassiopeia/scss/vendor/choicesjs/choices.scss

```diff
@@ -98,7 +113,7 @@
     margin-bottom: 0;
     margin-left: 8px;
     line-height: 1;
-    border-left: 1px solid hsl(187, 100%, 32%);
+    border-left: 1px solid #ffffff;
```

Comment:
Why using `#ffffff` here? Elsewhere we use `$white`. Or isn't that the same?

Reply:
fixed